### DISCUSSION TITLE
Concurrent Wi-Fi client + AP D32PRO

### DISCUSSION THREAD

**Discussion by marcobrianza** [forkineye/ESPixelStick#801] [General]
I'm using a D32PRO with the latest CI release :
FW Version 4.0-ci11448233108
Build Date Oct 21 2024 - 20:39:03
Architecture ESP32
Board Named 32_pro

Can this board be connected to wifi and behave as Wi-Fi AP?
if yes what are the correct settings.

thank you
Marco

**Comment by MartinMueller2003** [comment]
Sorry missed this. Yes the ESPS has an AP mode. you can configure the SSID and the passphrase for WiFi on the flash page or use the defaults and modify them after the ESP boots. By default, the ESPS will come up in AP mode if it fails to connect to a local AP.
